sql/item_mods.sql

```diff
@@ -43576,8 +43576,24 @@ INSERT INTO `item_mods` VALUES (27991,30,10); -- 10 Macc
 INSERT INTO `item_mods` VALUES (27991,68,22); -- 22 Eva
 INSERT INTO `item_mods` VALUES (27991,31,37); -- 37 Meva
 INSERT INTO `item_mods` VALUES (27991,29,3); -- 3 Mdef
-INSERT INTO `item_mods` VALUES (27991,384,30); -- 3% Gear Haste
+INSERT INTO `item_mods` VALUES (27991,384,31); -- 3% Gear Haste
```

Comment:
we don't round up, we truncate. 3% of 1024 is 30.72 so 30 was correct until someone redone haste to be non stupid.

Reply:
dammit zyn

Reply:
@zynjec you get to fix it now

Reply:
30 divided by 1024 is 0.029296875. so 2.9% which will get truncated down to 2%.

Reply:
that's in reverse.

https://www.google.com/search?hl=en&q=whats%203%25%20of%201024

Reply:
He's off the hook because a ton of wrong values have been merged and on top of that at least a partial core change looks to have happened but nobody updated the values -_- (so we have values that were wrong before and after said change.. Enjoy those 10x hastes while they last kids)